domains/vishwaoffl.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "vishwavinoth",
+        "email": "rvishwavinoth@gmail.com"
+    },
+
+    "record": {
+        "A": ["vishwa"]
```

Comment:
Invalid A record, must be array of strings containing valid IPv4 records.

Reply:
"A" means what? feel free to guide me get the domain